bottles/backend/wine/winecommand.py

```diff
@@ -234,19 +232,21 @@ def get_env(self, environment: dict = None, return_steam_env: bool = False, retu
             ]
             gst_libs = [
                 "lib/gstreamer-1.0",
-                "lib32/gstreamer-1.0",
-                "lib64/gstreamer-1.0"
+                "lib32/gstreamer-1.0"
             ]
             
         for lib in runner_libs:
             _path = os.path.join(runner_path, lib)
             if os.path.exists(_path):
                 ld.append(_path)
 
-        for lib in gst_libs:
-            _root = os.path.basename(os.path.join(runner_path, lib))
-            if os.path.exists(os.path.join(runner_path, lib)):
-                env.add("GST_PLUGIN_SYSTEM_PATH", _root)
+        # Embedded GStreamer environment variables
+        if params["embedded_gstreamer"] and not return_steam_env:
```

Comment:
this param doesn't exists in the bottle configuration

Reply:
I added it, as it seemed interesting to have

Reply:
Ok but it should be added to the samples as well https://github.com/bottlesdevs/Bottles/blob/main/bottles/backend/models/samples.py

Reply:
Oh sorry, will do!

Is it needed at other places?

Reply:
No, this is the only, in this way Bottles will migrate the configurations to the new layout on next start

Reply:
Okay thank you.

With the commit you did before, I would assume you wanted it to be default? I'm not sure what would be the best setting for it

Reply:
Also, it seems that they are default parameters in https://github.com/bottlesdevs/Bottles/blob/main/bottles/backend/models/bottle.py, should it also be there?

Reply:
> Also, it seems that they are default parameters in https://github.com/bottlesdevs/Bottles/blob/main/bottles/backend/models/bottle.py, should it also be there?

That class is not yet used in Bottles. 



> Okay thank you.
> 
> With the commit you did before, I would assume you wanted it to be default? I'm not sure what would be the best setting for it

It should be a default if doesn't cause any problem

Reply:
> That class is not yet used in Bottles.

Okay, I'm not touching it then.

> It should be a default if doesn't cause any problem

I have not tested extensively, but I'm *guessing* it shouldn't. At least it's easily disabled...
I've updated the original post with a screenshot of the setting description.